spec/rails_helper.rb

```diff
@@ -2,15 +2,15 @@
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 # Prevent database truncation if the environment is production
-abort("The Rails environment is running in production mode!") if Rails.env.production?
+abort('The Rails environment is running in production mode!') if Rails.env.production?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.